CMakeLists.txt

```diff
@@ -25,7 +25,7 @@ INCLUDE( ilcsoft_default_settings )
 
 FIND_PACKAGE( Marlin 1.0 REQUIRED ) # minimum required Marlin version
 INCLUDE_DIRECTORIES( ${Marlin_INCLUDE_DIRS} )
```

Comment:
Do we need to find Marlin if we only want streamlog?

Reply:
There are some includes that need Marlin:
https://github.com/iLCSoft/KiTrack/blob/marlin-libraries/include/KiTrack/SubsetSimple.h#L6 so it should be needed?